test/FedToken.js

```diff
@@ -0,0 +1,63 @@
+
+// Hardhat tests are normally written with Mocha and Chai.
+
+// We import Chai to use its asserting functions here.
+const { expect } = require("chai");
+const { ethers } = require("hardhat")
+
+describe("FedToken contract", function () {
+  // Mocha has four functions that let you hook into the the test runner's
```

Comment:
Unnecessary cruft leftover from Hardhat. We can remove all the comments that aren't relevant to this class

Reply:
sounds good